test/src/test/java/org/corfudb/integration/StreamIT.java

```diff
@@ -1,68 +1,53 @@
 package org.corfudb.integration;
 
 import org.corfudb.protocols.wireprotocol.ILogData;
+import org.corfudb.protocols.wireprotocol.LogData;
 import org.corfudb.runtime.CorfuRuntime;
 import org.corfudb.runtime.view.stream.IStreamView;
 import org.junit.Before;
+import org.junit.Test;
 
+import java.util.List;
 import java.util.Random;
 import java.util.UUID;
+import java.util.stream.Stream;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
 /**
  * A set integration tests that exercise the stream API.
  */
 
-public class StreamIT extends AbstractIT {
```

Comment:
im discarding the test related changes, I just reused that test to run on a live system